.github/workflows/x-keycloak-operator.yml

```diff
@@ -146,3 +146,87 @@ jobs:
       - name: Show Container tags
         run: |
           echo "${{ steps.meta.outputs.tags }}" >> $GITHUB_STEP_SUMMARY
+
+      - name: Install Yq
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: sudo snap install yq
+
+      - name: Install OLM
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: ./operator/scripts/install-olm.sh
+
+      - name: Fetch previous version of Keycloak
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        # Could be used version of the last tag from the keycloak-rel/community-operators repo
+        run: echo ::set-output name=previous_tag::$(git describe --tags $(git rev-list --tags --max-count=1))
```

Comment:
We need the previous version of the KC operator in order to explicitly define in as `replaces` parameter in the KC CSV.
ATM, it fetches the last release tag of the Keycloak repository(not pre-release; nightly). 

At the end of this workflow, the new version of the OLM Bundle is tagged and pushed to the forks. The version could be fetched from there afterward.

Reply:
Please see my review comments.